### ISSUE TITLE
Change in appointment times

### ISSUE THREAD

**Issue by emanguy** [emanguy/CMSC331-Project2#25]
Appointments can only range between 8AM and 4 PM Mon-Fri according to Lupoli. We should update this in Results3.

**Comment by tbrandt1** [comment]
8? huh, it got extended by an hour then.

On Tue, Apr 28, 2015 at 12:40 AM, Evan Rittenhouse <notifications@github.com

**Comment by emanguy** [comment]
Yep. Shouldn't be hard to fix, just add a couple times in the times array
toward the beginning of Results3.

**Comment by jadedtdt** [comment]
also doing this in my work.

**Comment by jadedtdt** [comment]
```
    $printables = array(
    0 => "08:00 AM",
    1 => "08:30 AM",
    2 => "09:00 AM", 
    3 => "09:30 AM",
    4 => "10:00 AM", 
    5 => "10:30 AM",
    6 => "11:00 AM", 
    7 => "11:30 AM",
    8 => "12:00 PM", 
    9 => "12:30 PM",
    10 => "01:00 PM",
    11 => "01:30 PM", 
    12 => "02:00 PM", 
    13 => "02:30 PM",
    14 => "03:00 PM", 
    15 => "03:30 PM");
```

I didn't have to use the other array yet so you'll still have to fix that one

**Comment by emanguy** [comment]
Closed as of #34
